RecoEgamma/ElectronIdentification/python/Identification/mvaElectronID_Fall17_iso_V1_cff.py

```diff
@@ -9,17 +10,10 @@
 # for specific working points, and configure those cuts in VID
 #
 
-# This MVA implementation class name
-mvaFall17ClassName = "ElectronMVAEstimatorRun2Fall17Iso"
```

Comment:
Okay, this time the reason why everything failed is not so obvious to me anymore.

I am investigating, but I think it's because some other configuration fragments might rely on these kind on mvaClassName strings. I removed them in the individual MVA config files and instead put a single `mvaClassName = "ElectronMVAEstimatorRun2"` in [https://github.com/guitargeek/cmssw/blob/ElectronID_MVA2017_V2_102X/RecoEgamma/ElectronIdentification/python/Identification/mvaElectronID_tools.py](url), because all Electron Run2 MVAs now use the same class anyway (...and the `Fall17Iso` part moves to the mvaTag, resulting in the same valueMap name).

I will follow up soon.

Reply:
Just in case http://cms-sw.github.io/PRWorkflow

```
Run a basic battery of tests

runTheMatrix.py -l limited -i all
```

please update only after these tests pass in your local release.

Reply:
Yes that's what I will do! Sorry for all the testing, I didn't expect these complications...